docs/getting_started/features/accesibility.md

```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
 |----------------------------------|-------------------|
-|[stops.txt](/schedule/reference/#stopstxt)|`tts_stop_name` |
+|[stops.txt](/documentation/schedule/reference/#stopstxt)|`tts_stop_name` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table provides a readable version of the stop name, allowing text-to-speech tools to read the name aloud.
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#stopstxt"><b>stops.txt</b></a> <br>
+        </p>
+
+        | stop_id | stop_name  | stop_lat    | stop_lon   | tts_stop_name            |
+        |---------|------------|-------------|------------|--------------------------|
+        | TAS001  | 5 Av/53 St | 45.5035680  | -73.587079 | 5th avenue and 53 street |
 
 ## Route Colors
 
 Using route colors allows to accurately depict and communicate the color scheme assigned to specific routes by the agency’s design guidelines, this enables users to easily identify transit services by their official color.
 
+**Pre-requirements**: Implement Base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[routes.txt](/schedule/reference/#routestxt)|`route_color`, `route_text_color` |
+|[routes.txt](/documentation/schedule/reference/#routestxt)|`route_color`, `route_text_color` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table assigns an orange color to route RA using the HEX color code D95700, while also specifying that text should be rendered black.
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#routestxt"><b>routes.txt</b></a> <br>
+        </p>
+
+        | route_id | agency_id | route_short_name | route_long_name    | route_type | route_color | route_text_color |
+        |----------|-----------|------------------|--------------------|------------|-------------|------------------|
+        | RA       | agency001 |               17 | Mission - Downtown |          3 | D95700      |                0 |
 
 ## Bike Allowed
 
 Specifying bike allowance allows to indicate if vehicles serving specific trips are able to accommodate bicycles or not, helping users to plan and access services that enable them to make multimodal trips.
 
+**Pre-requirements**: Implement Base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[trips.txt](/schedule/reference/#tripstxt)|`bikes_allowed` |
+|[trips.txt](/documentation/schedule/reference/#tripstxt)|`bikes_allowed` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table specifies that the vehicle used in trip `AWE1` is able to accommodate at least one bicycle on board, while also specifying that bicycles cannot be accommodated in the vehicle used for trip `AWE2`.
```

Comment:
I am probably missing nuances of what "while" means in English here, but for me it implies a duration (it could be replaced by "during this time,). Also, having the first portion of the sentence with "the vehicle" as a subject and the second part with "the bike" as the subject can imply these are two different things.

Reply:
Here is a suggestion to make this sentence simpler:
```suggestion
 The following table specifies that the vehicle used in trip `AWE1` can accommodate at least one bicycle on board, and the vehicle used in trip `AWE2` cannot.
```